celext/lib.go

```diff
@@ -378,12 +378,20 @@ func (l lib) uniqueMemberOverload(itemType *cel.Type, overload func(lister trait
 }
 
 func (l lib) uniqueScalar(list traits.Lister) ref.Val {
-	exist := make(map[ref.Val]struct{})
-	for i := int64(0); i < list.Size().Value().(int64); i++ {
-		if _, ok := exist[list.Get(types.Int(i))]; ok {
+	size, ok := list.Size().Value().(int64)
+	if !ok {
+		return types.UnsupportedRefValConversionErr(list.Size().Value())
```

Comment:
Don't think this is really reachable but golangci-lint complained so I added error handling.